wallet/wallet.go

```diff
@@ -4594,20 +4577,20 @@ func (w *Wallet) SignTransaction(ctx context.Context, tx *wire.MsgTx, hashType t
 			}
 			if err != nil {
 				multisigNotEnoughSigs := false
-				class, addr, _, _ := txscript.ExtractPkScriptAddrs(
-					0,
-					additionalPrevScripts[txIn.PreviousOutPoint],
-					w.ChainParams(), true) // Yes treasury
-
-				if errors.Is(err, txscript.ErrInvalidStackOperation) &&
```

Comment:
I don't know what this was after yet since I haven't looked at the context closely enough.